### PR TITLE
REP-584: Loosening validation for Tiktok; cleaning up unit tests.

### PR DESCRIPTION
**Pull request by rajivraman**
@jec3 I noticed that you removed the parameter restrictions in lookupSocial(). So I did the same thing, rather than altering addSocial() to handle Tiktok. Also cleaned up some unit-test crud.

### CODE REVIEW

**Comment by dchancogne** [comment]
@rajivraman Can we fix the CI issues?

**Comment by rajivraman** [comment]
@dchancogne tests have been broken for a while mainly due to broken api keys: https://traackr.atlassian.net/browse/BI-8086

File: test/Influencers/InfluencersTest.php [hunk 1]
```diff
@@ -382,11 +382,55 @@ public function testLookupSocialInstagram()
 
     /**
      * @group read-only
-     * @expectedException \UnexpectedValueException
+     */
```
Comment (rajivraman) [inline test/Influencers/InfluencersTest.php:385]: Added a test for the Tiktok. Tested locally to make sure it works against dev.

File: test/Influencers/InfluencersTest.php [hunk 2]
```diff
@@ -843,7 +887,7 @@ public function testSearchWithAudienceParameter()
                 'network' => 'twitter',
                 'filters' => [
                     [
```
Comment (rajivraman) [inline test/Influencers/InfluencersTest.php:889]: Leftover cleanup from the audience switchover.

**Comment by jec3** [comment]
> @jec3 I noticed that you removed the parameter restrictions in lookupSocial(). So I did the same thing, rather than altering addSocial() to handle Tiktok. Also cleaned up some unit-test crud.

Since the Search API itself validates the platform against its current list of supported platforms it makes sense to keep this codebase's methods a bit more evergreen.

**Comment by dchancogne** [comment]
@rajivraman Fixed the api key issue. But test data is not available in prod.

**Comment by rajivraman** [comment]
> > @jec3 I noticed that you removed the parameter restrictions in lookupSocial(). So I did the same thing, rather than altering addSocial() to handle Tiktok. Also cleaned up some unit-test crud.
> 
> Since the Search API itself validates the platform against its current list of supported platforms it makes sense to keep this codebase's methods a bit more evergreen.

@jec3 , sounds good to me. Just a bit of history on the type of validation we included in this client: general rule was to include the minimum amount of validation that the LiveDocs page would apply. So, since the LiveDocs page has a dropdown for certain parameters, we added corresponding validation here to match it (and of course to avoid the round-trip to Core, even if it has its own validation). Not sure about the future of this client, so an evergreen approach in instances like this sounds good to me.

**Pull request merged**